doc/release.md

```diff
@@ -63,6 +63,9 @@ to the beta Rust release. The remerge is then necessary, to make sure that the
 Clippy commit, that was used by the now stable Rust release, persists in the
 tree of the Clippy repository.
 
+To find out if this step is necessary run `git branch master --contains beta`.
```

Comment:
Not sure if that works differently in newer git versions. I'm on 2.17.1 and first had to do `git fetch <clippy-remote> && git checkout beta` before I could run that command. Without the checkout I was getting `error: malformed object name beta`. Should we add that as well?